.github/workflows/publish-sdist-wheels.yml

```diff
@@ -10,7 +10,7 @@ env:
   MACOSX_DEPLOYMENT_TARGET: '10.13'
   CIBW_BUILD_VERBOSITY: 2
   # Only build on Python 3.x
-  CIBW_BUILD: 'cp3?-*'
+  CIBW_BUILD: 'cp3?-* cp27-*'
```

Comment:
I tried before and this fails due to cibuildwheel version compatibility problems when running with 1.6.4. 
You should relax the version number to make it works probably. But this is a bit scary to have wheels made with different versions ....

Reply:
Ok.....
Let's drop python 2 instead...